pkg/virt-handler/virtwrap/api/schema.go

```diff
@@ -143,6 +168,29 @@ type DomainList struct {
 	Items    []Domain
 }
 
+func (in *DomainList) DeepCopyInto(out *DomainList) {
+	v, _ := model.Clone(in)
+	out, _ = v.(*DomainList)
```

Comment:
Could you use the cast version everywhere which will panic (`out = v.(*DomainList)`)?

Reply:
:+1: